application/src/main/java/org/togetherjava/tjbot/config/Config.java

```diff
@@ -23,16 +23,26 @@
     private final String databasePath;
     private final String projectWebsite;
     private final String discordGuildInvite;
+    private final String heavyModerationRolePattern;
+    private final String softModerationRolePattern;
+    private final String mutedRolePattern;
 
+    @SuppressWarnings("ConstructorWithTooManyParameters")
```

Comment:
Makes me again wonder why this isn't a record, would be 100x more maintainable.
But not for this PR

Reply:
yeah